robot-command/src/main/java/org/obolibrary/robot/ExplainCommand.java

```diff
@@ -0,0 +1,146 @@
+  /** Logger. */
+  private static final Logger logger = LoggerFactory.getLogger(ExplainCommand.class);
+
+  /** Store the command-line options for the command. */
+  private Options options;
+
+  public ExplainCommand() {
+    Options o = CommandLineHelper.getCommonOptions();
+    o.addOption("i", "input", true, "load ontology from a file");
+    o.addOption("I", "input-iri", true, "load ontology from an IRI");
+    o.addOption("r", "reasoner", true, "reasoner to use: ELK, HermiT, JFact");
+    o.addOption("a", "axiom", true, "the axiom to explain");
+    o.addOption("m", "max", true, "the maximum number of explanations to retrieve");
+    o.addOption("f", "format", true, "the explanation result format: MD, OFN, etc.");
+    o.addOption("o", "output", true, "save explanation to a file");
+    options = o;
+  }
+
+  @Override
+  public String getName() {
+    return "explain";
+  }
+
+  @Override
+  public String getDescription() {
+    return "explain derivation of an inferred axiom";
+  }
+
+  @Override
+  public String getUsage() {
+    return "robot explain --input <file> --reasoner <reasoner> --axiom <axiom> <output>";
+  }
+
+  @Override
+  public Options getOptions() {
+    return options;
+  }
+
+  /**
+   * Handle the command-line and file operations for the ExplainOperation.
+   *
+   * @param args strings to use as arguments
+   */
+  @Override
+  public void main(String[] args) {
+    try {
+      execute(null, args);
+    } catch (Exception e) {
+      CommandLineHelper.handleException(e);
+    }
+  }
+
+  @Override
+  public CommandState execute(CommandState state, String[] args) throws Exception {
+    CommandLine line = CommandLineHelper.getCommandLine(getUsage(), getOptions(), args);
+    if (line == null) {
+      return null;
+    }
+    if (state == null) {
+      state = new CommandState();
+    }
+    IOHelper ioHelper = CommandLineHelper.getIOHelper(line);
+    state = CommandLineHelper.updateInputOntology(ioHelper, state, line);
+    OWLOntology ontology = state.getOntology();
+    QuotedEntityChecker checker = new QuotedEntityChecker();
+    checker.setIOHelper(ioHelper);
+    checker.addProvider(new SimpleShortFormProvider());
+    checker.addProperty(OWLManager.getOWLDataFactory().getRDFSLabel());
+    if (ontology != null) {
+      checker.addAll(ontology);
+    }
+    int max = 1;
+    if (line.hasOption("max")) {
+      max = Integer.parseInt(line.getOptionValue("max"));
+    }
+    OWLReasonerFactory reasonerFactory = CommandLineHelper.getReasonerFactory(line, true);
+    ManchesterOWLSyntaxInlineAxiomParser parser =
+        new ManchesterOWLSyntaxInlineAxiomParser(OWLManager.getOWLDataFactory(), checker);
+    String expression = line.getOptionValue("axiom");
+    OWLAxiom axiom = parser.parse(expression);
+
+    Set<Explanation<OWLAxiom>> explanations =
+        ExplainOperation.explain(axiom, ontology, reasonerFactory, max);
+
+    if (line.hasOption("output")) {
```

Comment:
Could you output the explanations as plain text to the console if no `--output` is provided, or should this be required? Alternatively, this could be `--explanations` instead and the output could be the subset of the ontology created below.